db/migrate/20160429133701_create_games.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.string :status
+      t.integer :moves
+      t.text :board
```

Comment:
{
  "row0": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row1": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row2": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row3": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row4": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row5": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row6": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row7": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row8": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}],
  "row9": [{...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}, {...}]
}

Where each of the objects inside the array looks something like this {"color": "white", "type": "pawn"}